### ISSUE TITLE
RTC Support

### ISSUE THREAD

**Issue by andreeaflorescu** [rust-vmm/vmm-reference#128]
Add support for the RTC. Without the RTC the time of the guest would be 1970. To add the RTC device, we need to create an RTC wrapper over the vm-superio/RTC device for which to implement the MMIO bus, and we need to add its configuration to the FDT. Since the RTC device implementation in rust-vmm is minimal, no interrupt handling is needed.

Depends on: #125

**Comment by andreeaflorescu** [comment]
This is now resolved. The RTC device is added to the vmm-reference, but the slot (MMIO region + IRQ) is hard coded. We should tackle this in a separate issue.
